spec/std/crypto/bcrypt/password_spec.cr

```diff
@@ -20,6 +20,12 @@ describe "Crypto::Bcrypt::Password" do
     it "parses digest" do
       password.digest.should eq("weXJt7sno2HdPVrMvVf06kGgAZvPkga")
     end
+
+    it "validates the hash string has the required amount of parts" do
+      expect_raises(Crypto::Bcrypt::Error, /Invalid hash string/) do
```

Comment:
```suggestion
      expect_raises(Crypto::Bcrypt::Error, "Invalid hash string") do
```

Reply:
@miketheman Any chance you might be able to apply this suggestion?

Reply:
@sija Thanks! @straight-shoota done!